hledger-lib/Hledger/Data/Types.hs

```diff
@@ -733,18 +735,31 @@ nullaccountdeclarationinfo = AccountDeclarationInfo {
 
 -- | An account, with its balances, parent/subaccount relationships, etc.
```

Comment:
Perhaps we should change this to "An account within a hierarchy", or something. Representing/navigating a hierarchy of account balances is Account's purpose.

Reply:
Done